tests/e2e-leg-5/istio/10-deploy-operator.yaml

```diff
@@ -0,0 +1,21 @@
+# (c) Copyright [2021-2023] Open Text.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- command: sh -c "cd ../../.. && make undeploy-operator || true"
```

Comment:
Now that one operator instance can manage multiple vdb instances why isn't this a problem?

Reply:
e2e-leg-5 is special. It only ever runs 1 testcase at a time. All of the tests that need to run the operator in a special way (e.g. webhook disabled, helm overrides, etc.) are in this leg.